lib/web/ui/cluster.go

```diff
@@ -69,6 +69,25 @@ func NewClusters(remoteClusters []reversetunnel.RemoteSite) ([]Cluster, error) {
 	return clusters, nil
 }
 
+// NewClustersFromRemote creates a slice of Cluster's, containing data about each cluster.
+func NewClustersFromRemote(remoteClusters []services.RemoteCluster) ([]Cluster, error) {
+	clusters := []Cluster{}
+	for _, rc := range remoteClusters {
+		cluster := Cluster{
+			Name:          rc.GetName(),
+			LastConnected: rc.GetLastHeartbeat(),
+			Status:        rc.GetConnectionStatus(),
+		}
+		clusters = append(clusters, cluster)
+	}
+
+	sort.Slice(clusters, func(i, j int) bool {
+		return clusters[i].Name < clusters[j].Name
+	})
```

Comment:
nit: this does not keep the root cluster first in the list

Reply:
We do client-side sorting so this is not really needed.